src/pages/index.tsx

```diff
@@ -1,31 +1,41 @@
 import React from "react";
+
 import { PageProps } from "gatsby";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
 
+import { IndexBestProducts } from "../components/index-page/IndexBestProducts";
 import { IndexBuiltForYou } from "../components/index-page/IndexBuiltForYou";
 import { IndexExasperation } from "../components/index-page/IndexExasperation";
 import { IndexHero } from "../components/index-page/IndexHero";
 import { IndexLaunched } from "../components/index-page/IndexLaunched";
 import { IndexLogos } from "../components/index-page/IndexLogos";
 import { IndexMoreMomentum } from "../components/index-page/IndexMoreMomentum";
+import { IndexOneSmallStep } from "../components/index-page/IndexOneSmallStep";
 import { IndexPartnership } from "../components/index-page/IndexPartnership";
 import { IndexShipFaster } from "../components/index-page/IndexShipFaster";
 import { IndexStopStressing } from "../components/index-page/IndexStopStressing";
 import { Layout } from "../components/layout/Layout";
 import "../css/index.css";
 
 const IndexPage = (pageProps: PageProps) => {
+  const queryClient = new QueryClient();
```

Comment:
we'll want this set up in the layout. I believe I did this for the email opt in, which is now in mainline.